playbooks/modules/smb.yml

```diff
@@ -8,7 +8,7 @@
     verify_ssl: false
     api_user: 'user'
     api_password: 'password'
-    share_name: 'test_sample_smb'
+    smb_name: 'test_sample_smb'
```

Comment:
revert these.

Reply:
Due to the same variable name as the parameter name, was getting a sanity error for this.
Errors were the following:

`var-naming[no-reserved]: Variables names must not be Ansible reserved names. (name)
playbooks/modules/smb.yml:11

var-naming[no-reserved]: Variables names must not be Ansible reserved names. (name)
playbooks/modules/snapshotschedule.yml:10

var-naming[no-reserved]: Variables names must not be Ansible reserved names. (name)
playbooks/modules/synciqreports.yml:13

var-naming[no-reserved]: Variables names must not be Ansible reserved names. (name)
playbooks/modules/synciqtargetreports.yml:13

var-naming[read-only]: This special variable is read-only. (role_name)
playbooks/modules/user.yml:[24](https://github.com/dell/ansible-powerscale/actions/runs/5656537467/job/15323781425?pr=59#step:8:25)`

Hence, I renamed this variable.